Permanent/Constants/Strings.swift

```diff
@@ -42,6 +42,11 @@ extension String {
     static var tokenIncorrect: String { return "TokenIncorrect".localized() }
     static var emailAlreadyUsed: String { return "EmailAlreadyUsed".localized() }
     static var invalidFields: String { return "InvalidFields".localized() }
+    static var invalidCsrf: String { return "InvalidCsrf".localized() }
+    static var incorectOldPassword: String { return "IncorectOldPassword".localized() }
```

Comment:
typo

Reply:
invalidCsrf was changed with generic errorMessage.

Reply:
incorect is written with one R instead of two R's "incorrect"

Reply:
resolved with commit [ba1008b](https://github.com/PermanentOrg/permanent-ios/pull/53/commits/ba1008b903c40e28cad6d9989b10ecfd124cd099)